compiler/rustc_ast/src/util/comments.rs

```diff
@@ -47,54 +46,66 @@ pub fn beautify_doc_string(data: Symbol) -> String {
             j -= 1;
         }
 
+        if i != 0 || j != lines.len() { Some((i, j)) } else { None }
+    }
+
+    /// remove whitespace-only lines from the start/end of lines
+    fn vertical_trim(lines: Vec<&str>, i: usize, j: usize) -> Vec<&str> {
         lines[i..j].to_vec()
     }
```

Comment:
I don't think we need this function at all - I would rather just use it inline. We can also get rid of the `to_vec` if you change `horizontal_trim` to take a slice instead, that removes another copy.

Reply:
Good catch!